ROADMAP.md

```diff
@@ -8,124 +7,164 @@ Notes: This document is still in review will may be heavily modified based on st
+The "Definition of Done" for projects/sub-projects that involve writing new protocols/ modify existing ones usually consist of the following:
+- If a specification change is required:
+    - [ ] Spec is merged and classified as "Candidate Recommendation"
+    - [ ] (by virtue of the above) At least one major reference implementation exists
+- [ ] A well established testing criteria is met (defined at the outset of the project including but not limited to testing via Testground, compatibility tests with other implementations in the Release process, etc.)
+- [ ] Public documentation (on docs.libp2p.io) exists
+
+Supporting projects (such as testing or benchmarking) may have different criteria.
+
+## Benchmarking and Testing
+As mentioned in our [vision](#vision), performance and stability are core libp2p tenets. Rigorous benchmarking and testing help us uphold them. Related projects are listed in the [libp2p/test-plans roadmap](https://github.com/libp2p/test-plans/pull/56) and the [testground/testground roadmap](https://github.com/testground/testground/blob/master/ROADMAP.md). Our major priorities in Q4’22 and Q1’23 are:
+- [interoperability testing](https://github.com/libp2p/test-plans/issues/53) (across implementations & versions and between transports, muxers, & security protocols)
+- performance [benchmark go-libp2p using Testground](https://github.com/testground/testground/pull/1425) (create a benchmark suite to run in CI, create a public performance dashboard, [demonstrate libp2p is able to achieve performance on par with HTTP](https://github.com/libp2p/test-plans/issues/27))
 
-Others supporting projects (like testing or benchmarking) will have different criteria.
+These projects are parallel workstreams, weighed equally with roadmap items in this document. Some efforts like interoperability testing have a higher priority than implementation projects. The go-libp2p co-owns these efforts with the js-libp2p, rust-libp2p, and Testground/IPDX teams.
 
+[**Click here to see the shared Q4’22/Q1’23 testing and benchmarking priorities.**](https://github.com/libp2p/test-plans/pull/56)
 
 ## 🛣️ Milestones
 ### 2022
 
 #### Early Q4 (October)
--   A.1⚡Handshakes at the Speed of Light - **[Early Muxer Negotiation](https://github.com/libp2p/specs/issues/426)**
-- E.1 🧪 Future-proof testing - **RTT test for Early Muxer Negotiation**
-
-#### Mid Q4 (November)
--   F.1 ⏱ Future-proof Benchmarking - **[Using nix-builders](https://github.com/testground/testground/pull/1425)**
+- [B.1 ⚡ Early Muxer Negotiation](#1-early-muxer-negotiation)
 
 #### End of Q4 (December)
--   B.1 🧠 Smart Dialing - **Happy Eyeballs**
--   C 📊 Comprehensive Metrics - **Swarm metrics**
--   D.1 📺 Universal Browser Connectivity - **[WebRTC Browser -> Server](https://github.com/libp2p/go-libp2p/pull/1655)**
+- [A.1 📺 WebRTC Browser -> Server](#1-webrtc-for-browser-to-server)
+- [C.1 🧠 Happy Eyeballs](#1-happy-eyeballs)
+- [D 📊 Swarm Metrics](#e--comprehensive-metrics)
```

Comment:
This will be great, but I question whether we can get these done if we're pouring gas on interoperability testing.

Reply:
True, in an internal discussion we had some hopes about who could be the DRI for these specifically D. https://www.notion.so/pl-strflt/Comprehensive-Metrics-3de671a657ff49908a307f69a63a64df
Which could make it feasible to hit this goal